changelogs/fragments/fix-world-readable-skip-message.yaml

```diff
@@ -0,0 +1,9 @@
+---
+bugfixes:
+- The fix for CVE-2018-10875 prints out a warning message about skipping
```

Comment:
can we link from the CVE name to a description?

Reply:
Would this one be good? https://access.redhat.com/security/cve/cve-2018-10875

I looked around and the mitre database of CVEs doesn't have a very good description.